src/packagedcode/podspec.py

```diff
@@ -0,0 +1,271 @@
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+import csv
+import glob
+import io
+import logging
+import os
+import re
+
+import attr
+from packageurl import PackageURL
+
+from commoncode.fileutils import py2
+from commoncode import filetype
+from commoncode import fileutils
+from packagedcode import models
+
+
+"""
+Handle Podspec Package Manager
+"""
+
+TRACE = False
+
+logger = logging.getLogger(__name__)
+
+if TRACE:
+    import sys
+    logging.basicConfig(stream=sys.stdout)
+    logger.setLevel(logging.DEBUG)
+
+
+@attr.s()
+class PodspecPackage(models.Package):
+    metafiles = ('*.podspec',)
+    extensions = ('.podspec',)
+    default_type = 'podspec'
+    default_primary_language = 'Swift'
+    default_web_baseurl = None
+    default_download_baseurl = None
+    default_api_baseurl = None
+
+    @classmethod
+    def recognize(cls, location):
+        yield parse(location)
+
+
+def is_podspec(location):
+    """
+    Checks is the file is a podspec file or not.
+    """
+    return (filetype.is_file(location) and location.endswith('.podspec'))
+
+
+def parse(location):
+    """
+    Return a Package object from a .podspec file or None.
+    """
+    if not is_podspec(location):
+        return
+
+    package_data = parse_podspec(location)
+    return build_package(package_data)
+
+
+def build_package(podspec_data):
+    """
+    Return a Pacakge object from a package data mapping or None.
+    """
+    name = podspec_data.get('name')
+    version = podspec_data.get('version')
+    description = podspec_data.get('description')
+    homepage_url = podspec_data.get('homepage')
+    authors = podspec_data.get('author') or []
+    email = podspec_data.get('email') or []
+
+    parties = []
+    if authors:
+        parties.append(
+            models.Party(
+                name=', '.join(authors),
+                email=', '.join(email),
+                role='author'
+            )
+        )
+
+    dependencies = podspec_data.get('dependencies')
+    package_dependencies = []
+    if dependencies:
+        for dep_name, dep_version in dependencies.items():
+            package_dependencies.append(
+                models.DependentPackage(
+                    purl=PackageURL(
+                        type='gem', name=dep_name).to_string(),
+                    scope='dependencies',
+                    is_runtime=True,
+                    is_optional=False,
+                    requirement=dep_version,
+                )
+            )
+    package = PodspecPackage(
```

Comment:
Why not collect license information in these? some of your test podspec files have them. IMO this is some of the most important pieces of data we want to collect

Reply:
I will add add code for this too. :)

Reply:
Code to collect license data is added. :)